lib/ngx/process.lua

```diff
@@ -0,0 +1,35 @@
+-- Copyright (C) Yichun Zhang (agentzh)
+
+
+local ffi = require 'ffi'
+local base = require "resty.core.base"
+local getfenv = getfenv
+
+
+local C = ffi.C
+local _M = { version = base.version }
+
+
+ffi.cdef[[
+void ngx_http_lua_ffi_worker_privileged(int enable);
```

Comment:
I finally decided to use `ngx_http_lua_ffi_privileged_agent`, I think the user can choose to enable or disable, free to switch.